<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>929</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>瓦妈妈数据库</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>1051</width>
      <height>861</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <column>
        <property name="text">
         <string>姓名</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>年龄</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>电话</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>声音类型</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>备注</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>上一页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>跳转至</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>下一页</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action111">
   <property name="text">
    <string>111</string>
   </property>
  </action>
  <action name="action111_2">
   <property name="text">
    <string>111</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
